scripts/dashboards_ui_lib.py

```
@@ -0,0 +1,389 @@
+        """,
+        element,
+        value,
+    )
+
+
+def sleep_if_needed(seconds: float) -> None:
+    if seconds > 0:
+        time.sleep(seconds)
+
+
+def run_index_pattern_creation_workflow(
```

Ruff (0.15.7)

[warning] Too many branches (16 > 12)

(PLR0912)